diagnostic.md

```diff
@@ -16,13 +16,13 @@ Graces-MacBook-Pro:try grace$
 
 What is the full path to the current directory?
 
-_Replace this text with your answer._
+/Users/grace/wdi/try/
 
 ## Question 2
 
 Describe GitHub, **briefly** in your own words.
 
-_Replace this text with your answer._
+A website where you can store versions of your code.  URLS are created for code which can be shared with others for review.
 
```

Comment:
Correct. You've got the right idea, but the machanics are a little different.

A user interface (UI) for accessing git's version control capabilities for our projcet repositories.